### ISSUE TITLE
Month Calendar view covers entire screen

### ISSUE THREAD

**Issue by wkeull** [MMM-CalendarExt2/MMM-CalendarExt2#93]
When using the below configuration:
					{
					        name: "View2",
					        mode: "month",
					        title: "All",
					        calendars: ["US holiday", "Personal"],
					        position: "bottom_bar",
					},
The month calendar is not bounded to the bottom_bar and completly covers the screen. Other views such as monthly are properly bounded.

**Comment by Kris1972** [comment]
I have a similar issue when trying to use "middle_center".

**Comment by Dwyerave** [comment]
I have the same issue

**Comment by su-do-rm-rf** [comment]
@wkeull , @Kris1972 , @Dwyerave 

Did any of you happen to find an answer to this?  I have the same problem.

**Comment by Kris1972** [comment]
@su-do-rm-rf  Nope. I ended up going with a totally different layout to make it work. Used a bunch of daily calendars in different locations only showing specific items. Sucks because I can only fit 2 days worth of content, where if I'd been able to center a weekly view and take up most of the screen I might have been able to fit it all in. I can't complain too much, as I'm not really using it as a Magic Mirror, but as a family calendar. Sorry!

**Comment by KristjanESPERANTO** [comment]
The `month` and `week` views are designed to be wide — they render a 7-column grid that expands to fill its container. When placed in `bottom_bar`, MagicMirror gives the module the full screen width, which is why it covers the screen. This is expected behaviour.

As noted in the [documentation](https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/main/docs/Configuration/Views/week%2C-month.md):

> `view:week` and `view:month` could be too wide and tall for small screen. Use it carefully.
> If you set these view positions as `fullscreen_below` or `fullscreen_above`, these views will be fixed in bottom.

For better control over the width, you can adjust the CSS custom properties in your `custom.css`:

```css
.CX2 {
  --column-max-width: 300px;
}
```

Closing this as the behaviour is by design. Feel free to reopen if you have further questions!
